analytics.tf

```
@@ -0,0 +1,113 @@
+resource "aws_s3_bucket" "dynamodb_s3export" {
+  bucket = "${var.name}-dynamodb-exports"
```

```suggestion
  bucket = "dynamodb-exports-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}"
```

(consistent w/ the other bucket, also adding region+accountid in the bucket name prevents collisions, since S3 buckets across accounts+regions live in a global namespace)